.gitignore

```diff
@@ -29,4 +29,5 @@ jetpack-*/bin
 # e2e outputs
 /test-results/
 cypress/results/
-e2eStorageState.json
\ No newline at end of file
+e2eStorageState.json
+/unique-index-name.php
```

Comment:
ES E2E tests add this plugin. I am adding it to `.gitignore` to avoid accidental commit.